### ISSUE TITLE
Compatibility Issue: FaceAiSharp DetectFaces Method Fails with ImageSharp 3.1.1 Due to Missing ImageInfoExtensions

### ISSUE THREAD

**Issue by EbinJohn** [georg-jung/FaceAiSharp#30]
# Compatibility Issue with ImageSharp 3.1.1: Missing 'ImageInfoExtensions'

## Description

When the FaceAiSharp library is used alongside the latest version of ImageSharp (3.1.1), the `DetectFaces` method throws a runtime exception, indicating a compatibility issue. This problem does not occur with the older version of ImageSharp (2.1.5), suggesting that changes in ImageSharp's library, specifically the removal or modification of `ImageInfoExtensions`, might be the cause.

## Environment

- **.NET Runtime**: .NET 6
- **FaceAiSharp Version**: Latest
- **ImageSharp Version**: 3.1.1
- **Operating System**: Windows 11

## Exception Details

Could not load type 'SixLabors.ImageSharp.ImageInfoExtensions' from assembly 'SixLabors.ImageSharp, Version=3.0.0.0, Culture=neutral, PublicKeyToken=d998eea7b14cab13'.

## Steps to Reproduce

1. Ensure the .NET 6 environment is set up with the latest versions of FaceAiSharp and ImageSharp (3.1.1).
2. Run the following code, replacing file paths with valid image paths on your system:

```csharp
var selfiePath = @"<ReplaceWithValidPath>\Selfie.jpg";
var groupPhotoPath = @"<ReplaceWithValidPath>\GroupPhoto.jpg";

// Load images from file paths
using var selfieStream = File.OpenRead(selfiePath);
var selfie = Image.Load<Rgb24>(selfieStream);

using var groupPhotoStream = File.OpenRead(groupPhotoPath);
var groupPhoto = Image.Load<Rgb24>(groupPhotoStream);

var detector = FaceAiSharpBundleFactory.CreateFaceDetectorWithLandmarks();
var recognizer = FaceAiSharpBundleFactory.CreateFaceEmbeddingsGenerator();

var selfieFaces = detector.DetectFaces(selfie);
if (!selfieFaces.Any())
{
    Console.WriteLine("No faces detected in the selfie.");
    return;
}
```

## Expected Behavior

The `DetectFaces` method should successfully identify faces within the provided images without triggering any exceptions.

## Actual Behavior

An exception is thrown by the method, pointing to the inability to load the type `ImageInfoExtensions`, which suggests that this class has either been removed or relocated in the latest ImageSharp version.

## Possible Solution

A review of the changes made between ImageSharp versions 2.1.5 and 3.1.1 could clarify whether `ImageInfoExtensions` has been deprecated or replaced. Adapting FaceAiSharp to align with these updates could potentially resolve the issue.

**Comment by georg-jung** [comment]
Fixed by #28

**Comment by georg-jung** [comment]
An updated version is now available on NuGet: https://www.nuget.org/packages/FaceAiSharp/0.5.23
